src/components/main/flight/ReceiptWithFlightScreen.js

```diff
@@ -1,232 +1,213 @@
+        <View style={styles.innerView}>
+          <View style={styles.topInnerView}>
+            <Text style={styles.flightNumberLabel}>
+              Flight Number {""}
+              <Text style={{ fontFamily: "Poppins-bold" }}>
+                {flightChars} {flightNums}
+              </Text>
+            </Text>
+            <Text style={styles.header}>
+              The first step was taken. The rest is up to us!
+            </Text>
+            <Image source={leaf} style={styles.leaf_img} />
           </View>
-          <View style={styles.receiptContainer}>
-            <View style={styles.textRow}>
-              {/*Route*/}
-              <Text style={styles.receiptTextLeft}>{i18n.t('ROUTE')}</Text>
-              {/* Departure to Arrival */}
-              {tripIndex == 1 &&
-                <Text style={styles.receiptTextRight}>{depCityName} &#10230; {arrCityName}</Text>
-              }
-              {tripIndex == 2 &&
-                <Text style={styles.receiptTextRight}>{depCityName} &#10231; {arrCityName}</Text>
-              }
-            </View>
-            <Dash style={styles.dashedLine} dashColor={COLORS.lightgrey} dashGap={0} />
-            <View style={styles.textRow}>
-              {/*Carbon Footprints*/}
-              <Text style={styles.receiptTextLeft}>{i18n.t('CARBON FOOTPRINT (METRIC TONS)')}</Text>
-              <Text style={styles.receiptTextRight}>{footprint}</Text>
-            </View>
-            <Dash style={styles.dashedLine} dashColor={COLORS.lightgrey} dashGap={0} />
-            <View style={styles.textRow}>
-              {/*Total trees donated in transaction*/}
-              <Text style={styles.receiptTextLeft}>{i18n.t('TOTAL TREES')}</Text>
-              <Text style={styles.receiptTextRight}>{treeNum}</Text>
-            </View>
-            <Dash style={styles.dashedLine} dashColor={COLORS.lightgrey} dashGap={0} />
-            <View style={styles.textRow}>
-              {/*Years to neutralize*/}
-              <Text style={styles.receiptTextLeft}>{i18n.t('YEARS TO COMPENSATE')}</Text>
-              <Text style={styles.receiptTextRight}>{years}</Text>
-            </View>
-            <Dash style={styles.dashedLine} dashColor={COLORS.lightgrey} dashGap={0} />
-            <View style={styles.textRow}>
-              {/*Cost of transaction*/}
-              <Text style={styles.receiptTextLeft}>{i18n.t('PRICE')}</Text>
-              <Text style={styles.receiptTextRight}>${total_cost}</Text>
-            </View>
+          <View style={styles.bottomInnerView}>
+            <List>
+              {/* Flight Distance */}
+              <ListItem>
+                <Left>
+                  <Text style={styles.itemTitle}>Flight Distance (km)</Text>
+                </Left>
+                <Right>
+                  <Text style={styles.itemValue}>{distance}</Text>
+                </Right>
+              </ListItem>
+              {/* Carbon FootPrint */}
```

Comment:
Please make Carbon Footprint and (Metric Tons) two separate lines (still the same list item though)